os-autoinst-obs-auto-submit

```diff
@@ -128,6 +136,10 @@ for package in os-autoinst openQA os-autoinst-distri-opensuse-deps; do
         update_package $package
     )
 done
+if [[ $failed_packages ]]; then
+    echo "failed packages:$failed_packages"
+    false
+fi
 )
 
 rm -rf "$TMPDIR"
```

Comment:
To do what I suggested above I guess this should be put into a signal handler to be called on "EXIT" so that it's also done in case of error

Reply:
Yes, I'll do that as an individual commit.